.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/usr/local/miniconda2/bin/python"
```

Comment:
Do we really need this file in the repo?

Reply:
This should not be here, you are right!